Assets/Scripts/Managers/GameManager.cs

```diff
@@ -380,6 +372,14 @@ private void OnFadeIn()
             sceneTransitionCanvas.FadeIn -= OnFadeIn;
         }
 
+        /// <summary>
+        /// Returns true if the currently loaded scene is a cutscene or the main menu.
+        /// </summary>
+        public bool IsInCutsceneOrMainMenu()
+        {
+            return FindFirstObjectByType<CutsceneManager>() || SceneManager.GetActiveScene().name == "MainMenu";
+        }
+
```

Comment:
huh then why did we add main menu to cutscene list 🤔 

also would `FindAnyObjectByType` work?